ros2caret/verb/check_ctf.py

```diff
@@ -28,10 +27,33 @@ def add_arguments(self, parser, cli_name):
         parser.add_argument(
             'trace_dir', type=str,
             help='the path to the trace directory to be checked')
+        parser.add_argument(
+            '-m', '--max_callback_construction_order_on_path_searching',
+            type=int, dest='max_callback_construction_order_on_path_searching',
+            help='max construction order on path searching.'
+                 'The value must be positive integer. "0" is unlimited.',
```

Comment:
I suggest changing help message as below. If you agree this change, please apply same change to another files.

```suggestion
            help='callbacks whose construction_order are greater than this value are ignored on path searching.'
                 'The value must be positive integer or "0". "0" means unlimited.',
```

Reply:
consent